IDEAS/Buildings/Components/Window.mo

```diff
@@ -271,8 +269,9 @@ equation
   connect(shaType.hForcedConExt, radSolData.hForcedConExt) annotation (
     Line(points={{-68.5,-32.7043},{-76,-32.7043},{-76,-62.2},{-79.4,-62.2}},
                                                                     color = {0, 0, 127}));
-  connect(outsideAir.TDryBul_in, shaType.TDryBul) annotation(
-    Line(points = {{-42, -90}, {-46, -90}, {-46, -48}, {-58, -48}}, color = {0, 0, 127}));
+  connect(outsideAir.TDryBul_in, shaType.TDryBul) annotation (
+    Line(points={{-42,-90},{-46,-90},{-46,-49.4895},{-57.5,-49.4895}},
```

Comment:
```suggestion
    Line(points={{-42,-90},{-46,-90},{-46,-48},{-58,-48}},
```
Ideally, these types of changes are avoided... (at least, when they are not intended).

Reply:
I get your point, but avoiding this would require us to change models with a general text editor rather than Dymola. To be discussed during the next IDEAS meeting.